general/g.gui/main.c

```diff
@@ -58,7 +58,7 @@ int main(int argc, char *argv[])
     type->options = "wxpython,text,gtext";
     type->answer = "wxpython";
     type->guisection = _("Type");
-    
```

Comment:
Clean up the unrelated changes. I don't know if there is a way to catch these earlier more easily than just checking the changes in the PR online.

Reply:
Ok.